plansys2_pddl_parser/src/plansys2_pddl_parser/Ground.cpp

```diff
@@ -38,7 +38,7 @@ plansys2_msgs::msg::Node::SharedPtr Ground::getTree( plansys2_msgs::msg::Tree &
     node->name = name;
     for ( unsigned i = 0; i < params.size(); ++i ) {
         plansys2_msgs::msg::Param param;
-        if (i < replace.size()) {
+        if (i < replace.size() && params[i]<replace.size()) {
```

Comment:
I think linters are not working here, because you need spaces around the operator `<`

Reply:
Yeah, linters are deactivated: https://github.com/PlanSys2/ros2_planning_system/blob/ce2c82832e448ed0dde771d2ef50bcb206e88f82/plansys2_pddl_parser/CMakeLists.txt#L62-L63

I turned the linters on and fixed the lintering issues and got cpplint to pass, but I can't make both uncrustify and cpplint pass at the same time, there are some conflicts. I have no clue why this only happens in this package. Could there be a difference because the header files are `.h` instead of `.hpp`?

Reply:
Yeah, this problem was because the header files have the `.h` extension, I changed them to `.hpp` and fixed all lintering issues and activated the lintering test. I am not sure if it was the best approach to include these changes in this PR, let me know if you prefer me to create a separate PR.
@fmrico this PR should be ready for review now

Reply:
There were conflicts. I will open a new PR with the linter changes.